page-components/ArtistDrive/mocks.ts

```diff
@@ -0,0 +1,103 @@
+import type { Song } from 'types/model';
+
+export const mockSongList: Array<Song> = [
```

Comment:
i'll move this mock file to the /mocks/api file once this pr is merged

Reply:
KK